commonmark-ext-gfm-tables/src/main/java/org/commonmark/ext/gfm/tables/internal/HtmlTableNodeRenderer.java

```diff
@@ -8,12 +8,12 @@
 
 import java.util.*;
 
-public class TableNodeRenderer implements NodeRenderer {
+public class HtmlTableNodeRenderer implements NodeRenderer {
```

Comment:
Please name it `TableHtmlNodeRenderer` (keeping it consistent with `StrikethroughHtmlNodeRenderer`)